Monika-After-Story/MonikaModDev
# Use Fontgroup

#5572 #5566 

# Key Changes
* the standard interface font has been patched with m+ 2p and sourcehansansK/CN via FontGroup
* music menu font is now m+ 2p combined with sourcehansansK/CN
* new dev-only hotkeys for calendar:
    * `x` - decrement year by 1000
    * `c` - decrement year by 100
    * `v` - decrement year by 10
    * `b` - increment year by 10
    * `n` - increment year by 100
    * `m` - increment year by 1000

### Licensing
While SourceHanSans is listed in adobe fonts (which has very shitty license terms), the actual [github source](https://github.com/adobe-fonts/source-han-sans/tree/release) has the font under the SIL open font license, which means its safe to distribute.

# Testing
* Verify all eng text still looks okay
* Verify any text in jp/kr/simplified cn looks okay
* Verify entering jp/kr/simplified cn text as playername works and looks okay when playername/mcname is referenced elsewhere.
* Verify music menu looks okay when used with files that appear with jp/kr/simplified cn names
* verify calendar hotkeys only work in dev mode (`config.developer`)
